pkg/server/api_v2_databases_metadata.go

```diff
@@ -1011,3 +1103,7 @@ type tableMetadataWithDetailsResponse struct {
 	Metadata        tableMetadata `json:"metadata"`
 	CreateStatement string        `json:"create_statement"`
 }
+
+type dbMetadataForIdResponse struct {
```

Comment:
nit: I just have the same comment on this as the table details PR, where we might want to rename this to WithDetails instead of ForID.

Reply:
Done